src/services/__tests__/httpService.test.js

```diff
@@ -0,0 +1,82 @@
+import _ from 'lodash';
+
+import * as http from '../httpService';
+
+const defaultHeaders = { abc: 'def' };
+fetch = jest.fn().mockRejectedValue('error');
+const httpMethods = [http.remove, http.patch, http.post, http.put, http.get];
+
+describe('Http Service', () => {
+  afterEach(() => {
+    fetch.mockReset();
+    http.resetHeaders();
+  });
+
+  function getRequestOptions() {
+    return fetch.mock.calls[0][1];
+  }
+
+  function isUpdateMethod(method) {
+    return !(method.name === 'remove' || method.name === 'get');
+  }
+
+  httpMethods.map(method => {
+    test(`should set proper http method on ${method.name.toUpperCase()}`, async () => {
```

Comment:
i like that variables in the test name, never seen that before

Reply:
First time I've done it as well! The idea stemmed from a suggestion by @arkus7